src/app/public/modules/wait/wait.component.scss

```diff
@@ -57,19 +57,23 @@
 }
 
 @-webkit-keyframes sk-bounce {
-  0%, 100% {
+  0%,
+  100% {
```

Comment:
Is putting each CSS selector on a new line a standard? I don't recall this one.

Reply:
I don't believe we have a standard regarding this. I likely did it, because it looked more similar to our standards for other things like barrels and attributes